examples/custom_bindings/Cargo.toml

```diff
@@ -4,20 +4,14 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-egui = "0.32"
-eframe = { version = "0.31.0", default-features = false, features = [
-    "accesskit",     # Make egui compatible with screen readers. NOTE: adds a lot of dependencies.
-    "default_fonts", # Embed the default egui fonts.
-    "wgpu",          # Use the glow rendering backend. Alternative: "wgpu".
-    "persistence",   # Enable restoring app state when restarting the app.
-] }
-
-log = "0.4"
+egui = { workspace = true }
+eframe = { workspace = true }
+log = { workspace = true }
```

Comment:
```
thread 'main' panicked at /home/kemokempo/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/wgpu-25.0.2/src/api/instance.rs:117:13:
No wgpu backend feature that is implemented for the target platform was enabled. See `wgpu::Instance::enabled_backend_features()` for more information.
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

this is egui+wgpu bug but we need to add wgpu to each example as `wgpu = { workspace = true }`. Just placing it in workspace-dep is not enough for correct run

Reply:
done